core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -77,7 +77,7 @@
 
     private static final int NUM_CPU_CORES = Runtime.getRuntime().availableProcessors();
 
-    private static final String DEFAULT_VERBOSE_EXCEPTION_SAMPLER_SPEC = "rate-limited=10";
+    private static final String DEFAULT_VERBOSE_EXCEPTION_SAMPLER_SPEC = "rate-limit=10";
```

Comment:
rate-limiting?

Reply:
I think `rate-limit` is better than `rate-limiting` when used as a part of spec string since limit is a noun anyway.